examples/notebooks/mstl_decomposition.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Can you add an image for the examples directory and add this to the yaml file that lists notebooks?